charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -164,6 +164,7 @@ spec:
             privileged: true
             {{- else }}
             privileged: false
+            allowPrivilegeEscalation: false
```

Comment:
There are a bunch of containers in this `Daemonset`, please check if in any of the other container it is possible to set `allowPrivilegeEscalation: false`. From a quick glance I see we can add it to at lease `add-snat-rule-to-upstream-dns` and `network-unavailable-condition-ensurer`.